utils/migration_tools/model_registry_to_runs/README.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from the Model Registry to Experiments
+
+This script allows you to copy models and model versions from the Model Registry to Experiments, within the same project.
+
+## Why Migrate?
+Our revamped Web App adds a host of new features to Experiments, including better grouping, and enhanced reporting capabilities. By migrating your models and model versions to Experiments, you can take advantage of these features to better organize and compare your models.
+
+As a part of our focus on performance this year, the `Run` API is also be highly optimized for speed. Using the `Run` object instead of `Model` or `ModelVersion` objects will allow you to take advantage of these optimizations.
```

Comment:
```suggestion
As a part of our focus on performance this year, we have also highly optimized the `Run` API  for speed. Using the `Run` object instead of `Model` or `ModelVersion` objects will allow you to take advantage of these optimizations.
```